README.md

```diff
@@ -1,14 +1,56 @@
 # Reverse engineering Sony PlayMemories Camera Apps #
+
+| Latest release | Travis CI | AppVeyor |
+|--------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|------------------------------------------------------------------------|
+| [![Dowload](https://img.shields.io/github/downloads/ma1co/Sony-PMCA-RE/latest/total.svg)](https://github.com/ma1co/Sony-PMCA-RE/releases/latest) | ![Travis CI](https://img.shields.io/travis/ma1co/Sony-PMCA-RE.svg) | ![AppVeyor](https://img.shields.io/appveyor/ci/ma1co/Sony-PMCA-RE.svg) |
+
 The latest Sony cameras include an Android subsystem used to run apps from the proprietary Sony PlayMemories Camera App Store (PMCA). The tools provided in this repository allow you to install custom Android apps on your camera.
 
 Additional effort has been made to analyze the firmware update process. Using the obtained knowledge, we can execute custom code on almost all Sony Cybershot and Alpha cameras released between 2007 and 2016.
 
 ## Installing Android apps ##
-The list of cameras featuring the Android subsystem can be found [here](https://github.com/ma1co/OpenMemories-Framework/blob/master/docs/Cameras.md).
 
-The list of available apps can be found at [sony-pmca.appspot.com](https://sony-pmca.appspot.com/). If you are using Internet Explorer or Safari, apps can be installed directly from your browser. Other browsers and recent camera firmware updates are not compatible with this method anymore. It is recommended to use the native installers (pmca-gui and pmca-console) instead.
+### Using PMCGA-gui (recommended)
+
+#### Windows and OS X
+
+Download binaries for your system from the [release section](https://github.com/ma1co/Sony-PMCA-RE/releases/latest).
+
+#### Debian / Ubuntu
+
+Install the last release of PMCA-gui from the sources, using pip and virtualenv:
+
+```
+sudo apt install curl python-tk python-pip
```

Comment:
`sudo apt install curl python3-tk python-pip`

If you are using python3 this is how to install the python3-tk